src.ts/multisig.ts

```diff
@@ -0,0 +1,306 @@
+
+                return this.sendRawTransaction(tx, overrides).then((response) => {
+                    if (overrides && overrides.sendOnly) {
+                        return response;
+                    }
+                    let confirmations = (overrides && overrides.confirmations) ? Number(overrides.confirmations) : null;
+
+                    return this.signer.provider.waitForTransaction(response.hash, confirmations).then((receipt) => {
+                        if (1 == receipt.status) {
+                            return receipt;
+                        }
+                        throw this.signer.provider.checkTransactionReceipt(receipt, errors.CALL_EXCEPTION, "confirm multisig transaction failed", {
+                            method: "auth-signMutiSigTx",
+                            receipt
+                        });
+                    });
+                });
+            });
+        });
+    }
+
+    private sendRawTransaction(transaction: TransactionRequest, overrides?: any) {
+        if (!this.provider) { errors.throwError('missing provider', errors.NOT_INITIALIZED, { argument: 'provider' }); }
+
+        return populateTransaction(transaction, this.provider, this.address).then((tx) => {
+            return this.sign(tx, overrides).then((signedTransaction) => {
+                return this.provider.sendTransaction(signedTransaction, overrides).catch(error => {
+                    // Clear the cached nonce when failure happened to prevent it out of sequence
+                    this.clearNonce();
+                    throw error;
+                });
+            });
+        });
+    }
+
+     /**
+     * Create multisig wallet
+     * @param properties multisig properties
+     * @param signer signer wallet
+     * @param overrides options
+     */
+    static create(properties: MultiSigWalletProperties, signer: Signer, overrides?: any): Promise<TransactionResponse | MultiSigWallet> {
+        if (!Signer.isSigner(signer)) {
+            errors.throwError('create multisig wallet transaction require signer', errors.MISSING_ARGUMENT, { arg: 'signer' });
+        }
+        if (properties && 'string' === typeof (properties)) {
+            properties = JSON.parse(properties);
+        }
+
+        checkProperties(properties, {
+            owner: true,
+            threshold: true,
+            signers: true,
+        }, true);
+
+        return resolveProperties({ signerAddress: signer.getAddress() }).then(({ signerAddress }) => {
+            if (!signerAddress) {
+                errors.throwError('create multisig wallet require signer address', errors.MISSING_ARGUMENT, { required: 'signer' });
+            }
+            properties.owner = signerAddress; // Set signer address as owner
+
+            let multisig: MultiSigWalletProperties = checkFormat({
+                owner: checkString,
+                threshold: checkNumber,
+                signers: checkAny,
```

Comment:
The `signers` should be a list of address. Should we use the `arrayOf(checkAddress)`?